File: Angular/EstefaniasJeans/src/app/admin/pedido/model/PedidoDetalle.ts
import { Mercaderia } from "src/app/mercaderia/models/mercaderia";

export interface PedidoDetalle{
    Id_PedidoDetalle:string | null,
    Id_Pedido: string | null,
    Id_Mercaderia:string,
    Cantidad: number,
    Precio:number,
    PrecioTotal:number
    Mercaderia:Mercaderia | null
}

File: EstefaniasJeans/EstefaniasJeans/Data/Model/PedidoDetalle.cs
using System;

namespace EstefaniasJeans.Data.Model
{
  public class PedidoDetalle
  {
    public Guid Id_PedidoDetalle { get; set; }
    public Guid Id_Pedido { get; set; }
    public Guid Id_Mercaderia { get; set; }
    public int Cantidad { get; set; }
    public decimal Precio { get; set; }
    public decimal PrecioTotal { get; set; }
    public Mercaderia Mercaderia { get; set; }
    public Pedido Pedido { get; set; }
  }
}
